SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -37,25 +36,28 @@ public void SetEmployeeIdField(string storeEmployeeID)
         }
 
         /// <summary>
-        ///  Grabbing the Text from the ID in the search results to verify
+        /// Grabbing the text from ID in the search results
```

Comment:
"Gets the employee ID from search results"... follow this pattern in the other two methods below.